model/clusters_mgmt/v1/add_on_type.model

```diff
@@ -34,6 +34,9 @@ class AddOn {
 	// Indicates if this add-on can be added to clusters.
 	Enabled Boolean
 
+	// Indicates if this add-on is hidden.
```

Comment:
Maybe not the best description. :thinking:

Reply:
meh, looks fine to me